randovania/games/samus_returns/logic_database/header.json

```diff
@@ -5986,7 +5986,85 @@
             "to_shuffle_proportion": 1.0
         }
     },
-    "hint_feature_database": {},
+    "hint_feature_database": {
+        "chozo_seal": {
+            "long_name": "Chozo Seal",
+            "hint_details": [
+                "",
+                "near a Chozo Seal"
+            ]
+        },
+        "elevator": {
+            "long_name": "Elevator",
+            "hint_details": [
+                "",
+                "near an elevator"
+            ]
+        },
+        "teleporter": {
+            "long_name": "Teleporter",
+            "hint_details": [
+                "",
+                "near a teleporter"
```

Comment:
really not a fan of most of these right now... some of them are just so far away, that I don't think "near" is intuitive and that we should either use "in the same room as", or revamp the category